inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/ValidationUtils.java

```diff
@@ -43,4 +43,39 @@ public static void validate(Validator validator, Object object, Class<?>... grou
         }
     }
 
+    /**
+     * Extracts the hostname and validates the port from a JDBC URL with the specified prefix.
+     *
+     * @param fullUrl The full JDBC URL to extract the hostname and port from
+     * @param prefix  The expected prefix of the JDBC URL
+     * @throws Exception If the URL format is invalid or the port is invalid
+     */
+    public static void extractHostAndValidatePortFromJdbcUrl(String fullUrl, String prefix) throws Exception {
+        if (!fullUrl.startsWith(prefix)) {
+            throw new Exception("Invalid JDBC URL, it should start with " + prefix);
+        }
+        // Extract the host and port part after the prefix
+        String hostPortPart = fullUrl.substring(prefix.length() + 3);
```

Comment:
Suggest modifying the prefix to replace `+3`.

Reply:
Thanks for the correction, it has been replaced